MiniTeks/mks-server
# Add database server to store datapoints of mks-resourcesg

## Add redis-database to store and access mks-resources datapoints.

* database configurations in the mks-server
*  redis-client available to controllers
* kubernetes resources to access redis-db

#### How to test
1. Deploye redis-server - 
```bash
kubectl apply -f k8s/
```
2.
 ```bash
kubectl port-forward <your redis-db pod, e.g-mks-db-6f544776bf-lsp2r > 6379:6379
```
3. Build and execute (see README.md)
